components/navbar/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import classNames from './navbar.module.scss';
+
+const Navbar = () => {
```

Comment:
It can be directed exported from here
`export default const Navbar.....`

Reply:
can you explain it a little bit please. @swarajpure

Reply:
You are defining the function on line number 4, right?
Also, it's the **only** function in the file (that is the reason why you have used `default` while exporting it)

When it's the **only** function to be exported, we can directly export it while defining it on line number 4 like:
`export default const Navbar = () =>`

In the current case, `Navbar` is being referenced twice on line number 4 for defining it and on line number 20 for exporting, which is kind of unnecessary and doesn't make much sense.

I see that this repo has been following what you have done (probably you have referred existing code and made your component accordingly). Let us discuss this on Discord 👍🏻

Reply:
No need to do it as discussed on discord.